eosevm/assert.hpp

```diff
@@ -0,0 +1,2 @@
+#pragma once
+#define EOSEVM_ABORT(msg) silkworm::abort_due_to_assertion_failure(msg, __FILE__, __LINE__)
```

Comment:
The files in the `eosevm` directory should work in both EOS EVM Node and EOS EVM Contract, correct?

So we need to make this work with `eosio::check` in ANTELOPE as well.

Reply:
internally `silkworm::abort_due_to_assertion_failure` is already doing that